src/css/jwplayer/flags/floatingplayer.less

```diff
@@ -25,7 +25,7 @@
 
         .jw-flag-touch& {
             @media screen and (max-device-width : 480px) and (orientation: portrait) {
-                animation: jw-float-to-top 150ms cubic-bezier(0, 0.25, 0.25, 1) forwards 1;
+                animation: none;
```

Comment:
Is `animation: none` necessary? What animation are you overriding or cancelling here?

Reply:
https://github.com/jwplayer/jwplayer/blob/d45ac57c42f68649de32de20892913b68007676f/src/css/jwplayer/flags/floatingplayer.less#L11